lang/cs/TestRunner.proj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I had an impression that we want to run tests without rebuilding. That's what we did with vstest.console.exe call - I don't think it was ever rebuilding bits.

But we can add a dependency, of course. You mean something like this (for each .Tests project)?

    <ProjectReference Include="$(SolutionDir)\Org.Apache.REEF.Wake\Org.Apache.REEF.Wake.Tests.csproj">
      <Project>{214C64C6-04E5-4867-B69A-E3502EA50871}</Project>
      <Name>Org.Apache.REEF.Wake.Tests</Name>
    </ProjectReference>